.changeset/silly-crews-knock.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/langfuse': minor
+---
+
+Added support for custom headers in Langfuse requests.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)